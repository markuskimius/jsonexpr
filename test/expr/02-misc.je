PRINT(true ? 1234+2345 : 2345+3456);
PRINT(false ? 1234+2345 : 2345+3456);
