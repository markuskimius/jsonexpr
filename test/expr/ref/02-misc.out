3579
5801
